back/src/config/config.js

```diff
@@ -13,22 +13,25 @@ const client = new Client ({
     host: env.parsed.HOST,
     port: 5432,
     ssl: true
-  });
+});
 
+/**
+ * Connect database and execute queries
+ * @param {Array} queries - execution queries
+ */
 exports.connect = async function (queries) {
```

Comment:
This function executes any query that is sent to it. We should not name it connect.

Reply:
How about "executeQuery" ?